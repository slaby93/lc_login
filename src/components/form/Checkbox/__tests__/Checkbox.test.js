import React from 'react';
import { shallow } from 'enzyme';
import CheckboxStyled, { CheckboxInput } from './../Checkbox';

describe('<Checkbox />', () => {
  test('CSS', () => {
    const mountedComponent = shallow(<CheckboxStyled
      className="test"
      placeholder="test"
      field="test"
    >
      TEST
    </CheckboxStyled>);
    expect(mountedComponent).toMatchSnapshot();
  });

  test('HTML', () => {
    const mountedComponent = shallow(<CheckboxInput
      className="test"
      placeholder="test"
      field="test"
    >
      TEST
    </CheckboxInput>);
    expect(mountedComponent).toMatchSnapshot();
  });
});
