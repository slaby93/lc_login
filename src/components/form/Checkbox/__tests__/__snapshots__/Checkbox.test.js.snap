// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> CSS 1`] = `
.c0 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 15px 0;
}

.c0 .Elements__Label-eTDSKI {
  margin-right: 5px;
}

<CheckboxInput
  className="test c0"
  field="test"
  id={null}
  placeholder="test"
>
  TEST
</CheckboxInput>
`;

exports[`<Checkbox /> HTML 1`] = `
<div
  className="test"
>
  <Label
    for="test"
  >
    TEST
  </Label>
  <Checkbox
    field="test"
    id="test"
  />
</div>
`;
